### ISSUE TITLE
Фінансові програми, заявки: Картки

### ISSUE THREAD

**Issue by ndrhzn** [open-contracting/bi.dream.gov.ua#167]
**Джерело даних** - [Submissions endpoint](https://public-api.dream.gov.ua/marketplace/public/dream/submissions) 

**Ключові показники**

- **Кількість отриманих заявок / Number of applications received** - COUNT UNIQUE cdu_response/internal_code
- **Кількість відібраних заявок / Number of applications selected** - COUNT UNIQUE cdu_response/internal_code WHERE cdu_response/status="active"
- **Кількість заявників / Number of applicants** - COUNT UNIQUE cdu_response/parties/id WHERE cdu_response/parties/roles LIKE "%applicant%"
- **Кількість проєктів / Number of projects** - COUNT UNIQUE cdu_response/relatedProcesses/identifier WHERE cdu_response/relatedProcesses/relationship = "project"

**Comment by andrzejbeletsky** [comment]
чи правильно я розумію, що враховуємо заявки, які мають зв'язок з проєктами (через cdu_response/relatedProcesses/relationship = "project"). 
отже результати обраних фільтрів у проєктах вплинуть на кількість зв'язаних заявок.

**Comment by andrzejbeletsky** [comment]
картки готові
https://qsdemo.rbcgrp.com/sense/app/b0f13ce0-e592-48bc-9432-6da4de1f8cfb/sheet/4db2d41f-92d2-49db-9050-c9c37af50ccf/state/analysis

**Comment by ndrhzn** [comment]
@andrzejbeletsky так, через звʼязок заявок із проєктами, технічно результат обраних фільтрів щодо проєктів вплине на кількість заявок. поки можемо лишити це так, однак мусимо протестувати, чи це бажана поведінка з точки зору користувачів

**Comment by andrzejbeletsky** [comment]
done
take a look

**Comment by ndrhzn** [comment]
З картками наче все гаразд, але ймовірно справді треба буде переглянути трохи взаємодію карток із графіком #173
